airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/manifest.yaml

```diff
@@ -0,0 +1,2993 @@
+          value: >
+            {% if components_values.get("pivots") %}
+              type: NoPagination
+            {% else %}
+              type: DefaultPaginator
+              page_token_option:
+                type: RequestOption
+                inject_into: request_parameter
+                field_name: offset
+              pagination_strategy:
+                type: CursorPagination
+                cursor_value: "{% raw %}{{ response.get('offset', {}) }}{% endraw %}"
+                stop_condition: "{% raw %}{{ not response.get('offset', {}) }}{% endraw %}"
+            {% endif %}
+        # This mapping defines the configuration path for the stream.
+        # Property sets are stored as a list under the `source_config_1` key,
+        # which is the second entry in the stream configs list.
+        # Access other entries with the pattern `source_config_<index>`.
+        - type: ComponentMappingDefinition
+          field_path: ["retriever", "requester", "path"]
+          value: >
+            {% if components_values.get("pivots") %}
+            "properties/{{ components_values['source_config_1'][1] }}:runPivotReport"
+            {% else %}
+            "properties/{{ components_values['source_config_1'][1] }}:runReport"
+            {% endif %}
+        # This mapping defines path for the schema loader and paste property_id for each stream
+        - type: ComponentMappingDefinition
+          field_path: ["schema_loader", "retriever", "requester", "path"]
+          value: "properties/{{ components_values['source_config_1'][1] }}/metadata"
+        # NOTE: Yes, this is weird, but it exists solely for backward-compatibility.
+        # When the config contains multiple property IDs, it keeps the
+        # first stream as <report_name>, and names each additional stream
+        # <report_name>Property<property_id>.
+        - type: ComponentMappingDefinition
+          field_path: ["**", "name"]
+          value: "{{ components_values['name'] + 'Property' + components_values['source_config_1'][1] if components_values['source_config_1'][0] > 0 else components_values['name'] }}"
+        # This mapping add transformation that add property_id and date slice value to record
+        - type: ComponentMappingDefinition
+          field_path: ["transformations"]
+          create_or_update: True
+          value: >
+            {% set property_id = components_values['source_config_1'][1] %}
+              - type: AddFields
+                fields:
+                  - path: [ "property_id" ]
+                    value: {{ property_id }}
+                    value_type: string
+
+            {% set cohort = components_values.get("cohortSpec") %}
+            {% if not (cohort and cohort.get("enabled")) %}
+              - type: AddFields
+                fields:
+                  - path: [ "startDate" ]
+                    value: "{% raw %}{{ stream_slice.start_time or config['date_ranges_start_date'] }}{% endraw %}"
```

Comment:
can we add a comment about why we need `endraw`. I presume this is because we don't want to attempt interpolation on `stream_slice` since it doesn't exist in this context yet as we populate the template, but only when the sync is running,

Reply:
yes, I have added note